doc/source/flexibility/extended_mysqldump.rst

```diff
@@ -22,7 +22,7 @@ More information can be found on the :ref:`backup_locks` feature documentation.
 Compressed Columns support
 ==========================
 
-In |Percona Server| :rn:`5.7.17-11` :command:`mysqldump` has been extended to
+In |Percona Server| :rn:`5.7.17-11` :command:`mysqldump` was extended to
```

Comment:
Likewise (but comment out the whole session until the compressed columns are re-introduced in 8.0)